Documentation update against guidelines

DirkSonguer opened the pull request:
Overview
Documentation updates according to discussions with @thalbern .

Changes
- Changes according to writing style guideline 2: "Never write 'I'"
- Changes according to writing style guideline 3: "Avoid 'we'"
- Changing capitalization in headlines
- Rewrite of all UI component documentation to have unified structure. Also added more content.

DirkSonguer commented:
Ran docfx locally with resolved conflicts: Build succeeded.
        0 Warning(s)
        0 Error(s)

thalbern commented:
/azp run mrtk_docs

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 
-
-## Pressable Button
+## Pressable Button ##
```
```suggestion
## Pressable button ##
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
-
-## Pressable Button
+## Pressable Button ##
 See [Button](README_Button.md) page for the details.
```
```suggestion
See [button](README_Button.md) page for the details.
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 ![Hand Interaction Examples](../Documentation/Images/HandInteractionExamples/MRTK_Examples_PressTouch.png)
 
-## Bounding Box
+## Bounding Box ##
```
```suggestion
## Bounding box ##
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 
-## Bounding Box
+## Bounding Box ##
 See [Bounding Box](README_BoundingBox.md) page for the details.
```
```suggestion
See [bounding box](README_BoundingBox.md) page for the details.
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 ![Hand Interaction Examples](../Documentation/Images/HandInteractionExamples/MRTK_Examples_BoundingBox.png)
 
-## Manipulation Handler
+## Manipulation Handler ##
```
```suggestion
## Manipulation handler ##
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 
-## Manipulation Handler
+## Manipulation Handler ##
 See [Manipulation Handler](README_ManipulationHandler.md) page for the details.
```
```suggestion
See [manipulation handler](README_ManipulationHandler.md) page for the details.
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 
-## Slate
+## Slate ##
 See [Slate](README_Slate.md) page for the details.
```
```suggestion
See [slate](README_Slate.md) page for the details.
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 ![Hand Interaction Examples](../Documentation/Images/HandInteractionExamples/MRTK_Examples_Slate.png)
 
-## System Keyboard
+## System Keyboard ##
```
```suggestion
## System keyboard ##
```

thalbern commented on Documentation/README_HandInteractionExamples.md:
```diff
@@ -1,34 +1,33 @@
 
-## System Keyboard
+## System Keyboard ##
 See [System Keyboard](README_SystemKeyboard.md) page for the details.
```
```suggestion
See [system keyboard](README_SystemKeyboard.md) page for the details.
```

thalbern commented on Documentation/README_ObjectCollection.md:
```diff
@@ -1,33 +1,34 @@
+![Object collection](../Documentation/Images/ObjectCollection/MRTK_ObjectCollection_ExampleScene1.jpg)
 
-![Object Collection](../Documentation/Images/ObjectCollection/MRTK_ObjectCollection_3DObjects.jpg)
+[Periodic Table of the Elements](https://github.com/Microsoft/MRDesignLabs_Unity_PeriodicTable) is an example app that demonstrates how object collections work. It uses object collection to layout the 3D element boxes in different shapes.
```
```suggestion
[Periodic table of the elements](https://github.com/Microsoft/MRDesignLabs_Unity_PeriodicTable) is an example app that demonstrates how object collections work. It uses object collection to layout the 3D element boxes in different shapes.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -1,32 +1,31 @@
 ![Pointer](../Documentation/Images/Pointers/MRTK_Pointer_Main.png)
 
-A pointer is something attached to a controller that gives focus and dispatches inputs to the game object it is pointing to. Learn more about the MRTK input system [here](./Input/Overview.md), and a scripting/engineering overview of pointers [here](./Input/Pointers.md).
+A pointer is something attached to a controller that gives focus and dispatches inputs to the game object it is pointing to. Learn more about the MRTK input system [here](./Input/Overview.md), and a scripting / engineering overview of pointers [here](./Input/Pointers.md).
```
```suggestion
A [pointer](./Input/Pointers.md) is something attached to a controller that gives focus and dispatches [inputs]((./Input/Overview.md)) to the game object it is pointing to. 
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -1,32 +1,31 @@
+For a game object to be able to receive focus it must have a collider (so it can be hit by physics raycasts) and belong to one of the layers defined in the pointer raycast layer masks in the pointer profile.
 
-Pointers are instantiated automatically at runtime when a new controller is detected. The pointers that are created for each controller type are defined in the _Pointer Options_ in the _Pointer Profile_. You can have more than one pointer attached to a controller; for example, with the default pointer profile, WMR controllers get both a line and a parabolic pointer for normal selection and telepo ...
+Pointers are instantiated automatically at runtime when a new controller is detected. The pointers that are created for each controller type are defined in the *Pointer Options* in the *Pointer Profile*. You can have more than one pointer attached to a controller; for example, with the default pointer profile, WMR controllers get both a line and a parabolic pointer for normal selection and telepo ...
```
```suggestion
Pointers are instantiated automatically at runtime when a new controller is detected. The pointers that are created for each controller type are defined in the *pointer options* in the *pointer profile*. You can have more than one pointer attached to a controller; for example, with the default pointer profile, WMR controllers get both a line and a parabolic pointer for normal selection and teleportation respectively. Pointers communicate with each other to decide which one is active.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -1,32 +1,31 @@
+**GGV pointer**
 GGV stands for "Gaze, Gesture, Voice"<sup>[2](https://docs.microsoft.com/en-us/windows/mixed-reality/gaze)</sup>. The GGV pointer's position and direction is driven by the head's position and rotation. The pointer is used to provide input that matches the HoloLens V1 input style of head gaze + airtap<sup>[3](https://docs.microsoft.com/en-us/windows/mixed-reality/gestures)</sup>.
 
 In the pointer profile you can see that the V1 HoloLens input system is provided for you via the mapping of "GGVHand" (V1 HoloLens hand) to the GGVPointer. 
```
```suggestion
In the pointer profile you can see that the HoloLens V1 input system is provided for you via the mapping of "GGVHand" (V1 HoloLens hand) to the GGVPointer. 
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
-### Grab Pointer / Sphere Pointer
-<img src="../Documentation/Images/Pointers/MRTK_GrabPointer.jpg" width="600">
-
+**Grab pointer / Sphere pointer**
```
```suggestion
**Grab pointer / sphere pointer**
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
 
+Sphere pointer properties:
+* *Sphere Cast Radius*: The radius for the sphere used to query for grabbable objects.
+* *Debug Mode*: If true, draw the sphere that is used to query for grabbable objects
```
```suggestion
* *Debug Mode*: If true, draw the sphere that is used to query for grabbable objects.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
+* *debounceThreshold*: Once touching, the distance back hand needs to be pulled from surface before a touch release occurs.
 
 Other parameters:
+* *eventsToReceive*: If set to Touch, the object will receive _OnTouchDown_ _OnTouchUpdate_ and _OnTouchUp_ events. If set to pointer, the object will receive _OnPointerDown_ and _OnPointerUp_ events.
```
```suggestion
* *eventsToReceive*: If set to touch, the object will receive _OnTouchDown_ _OnTouchUpdate_ and _OnTouchUp_ events. If set to pointer, the object will receive _OnPointerDown_ and _OnPointerUp_ events.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
-#### Configuring NearInteractionTouchable
-When configuring your _Near Interaction Touchable_, make sure to configure the _localForward_ parameter to point out of the front of the button or other object you wish to make touchable. Also make sure that the touchable's _bounds_ matches the bounds of your touchable object. 
+Poke pointer properties:
+* *distBack*: How far input point can go behind surface before release event occurs
```
```suggestion
* *distBack*: How far input point can go behind surface before release event occurs.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
-When configuring your _Near Interaction Touchable_, make sure to configure the _localForward_ parameter to point out of the front of the button or other object you wish to make touchable. Also make sure that the touchable's _bounds_ matches the bounds of your touchable object. 
+Poke pointer properties:
+* *distBack*: How far input point can go behind surface before release event occurs
+* *distFront*: How far input point needs to be in front of surface before press event occurs
```
```suggestion
* *distFront*: How far input point needs to be in front of surface before press event occurs.
```

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -37,50 +36,41 @@ You can also simulate the V1 HoloLens GGV behavior on HoloLens 2 by mapping the
-- visuals - Game Object used to render finger tip visual (the ring on finger, by default)
-- line - optional line to draw from fingertip to the active input surface.
+**Mouse pointer**
+Mouse pointer is used for interacting with the mouse. This pointer will only be used if the active input is a mouse. By default, the MRTK profile does not provide a mouse as an input provider, as mouse input is instead used to simulate hand input. You may add a mouse in your MRTK profily by modifying the *Registered Service Providers* in your MRTK profile.
```
```suggestion
Mouse pointer is used for interacting with the mouse. This pointer will only be used if the active input is a mouse. By default, the MRTK profile does not provide a mouse as an input provider, as mouse input is instead used to simulate hand input. You may add a mouse in your MRTK profile by modifying the *Registered Service Providers* in your MRTK profile.
```

thalbern commented on Documentation/README_Solver.md:
```diff
@@ -1,40 +1,39 @@
+* **Orbital**: Locks to a specified position * offset from the referenced object.
+* **ConstantViewSize**: Scales to maintain a constant size relative to the view of the referenced object.
+* **RadialView**: Keeps the object within a view cone cast by the referenced object.
+* **SurfaceMagnetism**, casts rays to Surfaces in the world, and align the object to that surface.
```
```suggestion
* **SurfaceMagnetism**, casts rays to surfaces in the world, and align the object to that surface.
```

thalbern commented on Documentation/README_SystemKeyboard.md:
```diff
@@ -1,24 +1,28 @@
-# System Keyboard
-![System Keyboard](../Documentation/Images/SystemKeyboard/MRTK_SystemKeyboard_Main.png)
+# System Keyboard #
```
```suggestion
# System keyboard #
```

thalbern commented on Documentation/README_SystemKeyboard.md:
```diff
@@ -1,24 +1,28 @@
+# System Keyboard #
 
-Inside Unity app, you can invoke the system keyboard. HoloLens2's system keyboard supports direct hand interactions.
+![System Keyboard](../Documentation/Images/SystemKeyboard/MRTK_SystemKeyboard_Main.png)
```
```suggestion
![System keyboard](../Documentation/Images/SystemKeyboard/MRTK_SystemKeyboard_Main.png)
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
+![Tooltip](../Documentation/Images/Tooltip/MRTK_Tooltip_Main.png)
 
-In the scene file, you will be able to find various examples of ToolTip. First group on the left demonstrates the static ToolTip examples that are always visible. In the center, you can see the example of using multiple ToolTips on a single object. Each tooltip has different child object as a target object which works as an anchor. The group on the right shows the examples of dynamically spawning ...
+This example scene demonstrates an implementation of the ToolTip user interface element. Tool tips are usually used to convey a hint or extra information upon closer inspection of an object. Tool tips can be used to explain button inputs on the motion controllers or to label objects in the physical environment.
```
```suggestion
This example scene demonstrates an implementation of the tooltip user interface element. Tooltips are usually used to convey a hint or extra information upon closer inspection of an object. Tooltips can be used to explain button inputs on the motion controllers or to label objects in the physical environment.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
 
-## Directly adding to the scene and attaching to an object
-A ToolTip can be added directly to the Hierarchy and targeted to an object. To use this method:
+The [example scene](https://gfycat.com/WarmOblongBilby) demonstrates two ways to display a Tooltip on an object.
```
```suggestion
The [example scene](https://gfycat.com/WarmOblongBilby) demonstrates two ways to display a tooltip on an object.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
-- In the ToolTip prefab's Inspector panel, expand the Tool Tip (Script). Select a TipState and set other settings. 
-- Enter the ToolTip text in the Text field. 
-- Expand the ToolTipConnector (Script). Drag the object that is to have the ToolTip from the Hierarchy into the field labelled Target. This attaches the ToolToolTip connector to the object. 
+## How to use a tool tip ##
```
```suggestion
## How to use a tooltip ##
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
-- Enter the ToolTip text in the Text field. 
-- Expand the ToolTipConnector (Script). Drag the object that is to have the ToolTip from the Hierarchy into the field labelled Target. This attaches the ToolToolTip connector to the object. 
+## How to use a tool tip ##
+A tool tip can be added directly to the hierarchy and targeted to an object.
```
```suggestion
A tooltip can be added directly to the hierarchy and targeted to an object.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
+A tool tip can be added directly to the hierarchy and targeted to an object.
 
-This use of ToolTip assumes a ToolTip that is always showing or that is shown/hid in script by changing the TipState property of the ToolTip component.
+To use this method simply add a game object and one of the [tool tip prefabs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Tooltips) to the scene hierarchy. In the prefab's inspector panel, expand the *Tool Tip* (script). Select a tip state and configure the tool tip.  Enter the respective text for the tool tip in the  ...
```
```suggestion
To use this method simply add a game object and one of the [tooltip prefabs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Tooltips) to the scene hierarchy. In the prefab's inspector panel, expand the *Tool Tip* (script). Select a tip state and configure the tooltip.  Enter the respective text for the tool tip in the text field. Expand the *ToolTipConnector* (Script) and drag the object that is to have the tooltip from the hierarchy into the field labelled *Target*. This attaches the tooltip to the object. 
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
-This use of ToolTip assumes a ToolTip that is always showing or that is shown/hid in script by changing the TipState property of the ToolTip component.
+To use this method simply add a game object and one of the [tool tip prefabs](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Prefabs/Tooltips) to the scene hierarchy. In the prefab's inspector panel, expand the *Tool Tip* (script). Select a tip state and configure the tool tip.  Enter the respective text for the tool tip in the  ...
 
+This use assumes a tool tip that is always showing or that is shown / hidden via script by changing the tip state property of the tool tip component.
```
```suggestion
This use assumes a tooltip that is always showing or that is shown / hidden via script by changing the tooltip state property of the tooltip component.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
  
-## Dynamically spawning
-A ToolTip can be dynamically added to an object at runtime as well as pre-set to show and hide on a Tap or focus. Simply add the **ToolTipSpawner** script to any GameObject. In the script's Inspector, you can set delays for appearing and disappearing. You can also set a lifetime so that the ToolTip when spawned, will disappear after a duration. You can also set style properties such as Background ...
+## Dynamically spawning tool tips ##
```
```suggestion
## Dynamically spawning tooltips ##
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
-## Dynamically spawning
-A ToolTip can be dynamically added to an object at runtime as well as pre-set to show and hide on a Tap or focus. Simply add the **ToolTipSpawner** script to any GameObject. In the script's Inspector, you can set delays for appearing and disappearing. You can also set a lifetime so that the ToolTip when spawned, will disappear after a duration. You can also set style properties such as Background ...
+## Dynamically spawning tool tips ##
+A tool tip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspect ...
```
```suggestion
A tooltip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspector as well as a lifetime so that the tooltip will disappear after a set duration. Tooltips also feature style properties such as background visuals in the spawner script. By default the tooltip will be anchored to the object with the spawner script. This can be changed by assigning a GameObject to the anchor field.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
+## Dynamically spawning tool tips ##
+A tool tip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspect ...
 
+## Tool tips on motion controllers ##
```
```suggestion
## Tooltips on motion controllers ##
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
+A tool tip can be dynamically added to an object at runtime as well as pre-set to show and hide on a tap or focus. Simply add the [`ToolTipSpawner`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Tooltips/ToolTipSpawner.cs) script to any game object. Delays for appearing and disappearing can be set in the scripts inspect ...
 
+## Tool tips on motion controllers ##
+Tool tips can also be assigned to motion controllers, for example to explain the assigned actions of buttons. The example scene below also includes two tooltip groups on the bottom. These are layed out to match position of the buttons on the motion controllers. When motion controllers are detected, these tool tips will be attached automatically to the controllers, using [`AttachToController`](xre ...
```
```suggestion
Tooltips can also be assigned to motion controllers, for example to explain the assigned actions of buttons. The example scene below also includes two tooltip groups on the bottom. These are layed out to match position of the buttons on the motion controllers. When motion controllers are detected, these tooltips will be attached automatically to the controllers, using [`AttachToController`](xref:Microsoft.MixedReality.Toolkit.UI.Utilities.Solvers.AttachToController) script.
```

thalbern commented on Documentation/README_Tooltip.md:
```diff
@@ -1,39 +1,27 @@
-## Motion controller Tooltips
-The scene also includes two tooltip groups on the bottom. These ToolTips are layed out to match position of the buttons on the motion controllers. When motion controllers are detected, these ToolTips will be attached automatically to the controllers, using [`AttachToController`](xref:Microsoft.MixedReality.Toolkit.Utilities.Solvers.AttachToController) script.
+## Example scene ##
+In the [example scene files](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/UX/Tooltips/Scenes), you will be able to find various examples of tool tips. First group on the left demonstrates the static tool tips examples that are always visible. In the center, you can see the example of using multiple tool tips on a single object. ...
```
```suggestion
In the [example scene files](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Examples/Demos/UX/Tooltips/Scenes), you will be able to find various examples of tooltips. First group on the left demonstrates the static tooltips examples that are always visible. In the center, you can see the example of using multiple tooltips on a single object. Each tooltip has different child object as a target object which works as an anchor. The group on the right shows the examples of dynamically spawning tooltips.
```

julenka commented:
It looks like the mrtk_docs run is failing -- we should make sure that passes before we merge this in.

julenka commented:
It looks like there is an invalid cross reference:

[19-04-23 02:04:53.823]Warning:[BuildCommand.BuildCore.Build **Document.LinkPhaseHandlerWithIncremental.Apply Templates](Documentation/README_Tooltip.md)1 invalid cross reference(s) "Microsoft.MixedReality.Toolkit.UI.Utilities.Solvers.AttachToController".**

thalbern commented:
> It looks like there is an invalid cross reference:
> 
> (Documentation/README_Tooltip.md)1 invalid cross reference(s) "Microsoft.MixedReality.Toolkit.UI.Utilities.Solvers.AttachToController".

checking

thalbern commented:
> It looks like the mrtk_docs run is failing -- we should make sure that passes before we merge this in.

@DirkSonguer in your branch the offending xref should be Microsoft.MixedReality.Toolkit.Utilities.Solvers.AttachToController . seems like someone changed that namespace

DirkSonguer commented:
Thank you for the awesome and thorough review @thalbern! I added all your suggestions and will incorporate the new suggestions into the next pass of the documentation.

Also thank you @julenka for the hint, I updated the file reference.

thalbern commented:
/azp run mrtk_docs

thalbern commented:
@DirkSonguer there's another docs build break 
Build succeeded with warning.
[19-04-24 01:15:32.098]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.ConceptualDocumentProcessor.Save](Documentation/README_Pointers.md#L5)Invalid file link:(~/Documentation/(./Input/Overview.md)).
	1 Warning(s)
	0 Error(s) 

README_Pointers.md pointing to Overview.md

DirkSonguer commented:
/azp run mrtk_docs

thalbern commented:
/azp run mrtk_docs

thalbern commented:
Build succeeded with warning.
[19-04-24 03:07:17.275]Warning:[BuildCommand.BuildCore.Build Document.LinkPhaseHandlerWithIncremental.ConceptualDocumentProcessor.Save](Documentation/README_Pointers.md#L5)Invalid file link:(~/Documentation/(../Documentation/Input/Overview.md)).
	1 Warning(s)
	0 Error(s)

link is still broken

thalbern commented on Documentation/README_Pointers.md:
```diff
@@ -1,86 +1,76 @@
 ![Pointer](../Documentation/Images/Pointers/MRTK_Pointer_Main.png)
 
-A pointer is something attached to a controller that gives focus and dispatches inputs to the game object it is pointing to. Learn more about the MRTK input system [here](./Input/Overview.md), and a scripting/engineering overview of pointers [here](./Input/Pointers.md).
+A [pointer](../Documentation/Input/Pointers.md) is something attached to a controller that gives focus and dispatches [inputs]((../Documentation/Input/Overview.md)) to the game object it is pointing to. 
```
```suggestion
A [pointer](../Documentation/Input/Pointers.md) is something attached to a controller that gives focus and dispatches [inputs](../Documentation/Input/Overview.md) to the game object it is pointing to. 
```

thalbern commented:
/azp run mrtk_docs
